src/imitation/rewards/reward_nets.py

```diff
@@ -240,6 +239,46 @@ def __init__(
     def base(self) -> RewardNet:
         return self._base
 
+
+class ForwardWrapper(RewardNetWrapper):
```

Comment:
Do we still want to forward device and dtype? I could see this going either way. Subclasses should probably aim to make them consistent between `base` and the main class anyway I guess.

Reply:
Have addressed this.